### ISSUE TITLE
Is it possible to disable using Ansible on host even if it's installed?

### ISSUE THREAD

**Issue by int-ua** [geerlingguy/drupal-vm#596]
I don't really like installing custom roles with `sudo` and I had to remove Ansible for drupal-vm to move on, is there a better option?

**Comment by oxyc** [comment]
I believe it should work with:

```sh
mkdir -p provisioning/roles
ansible-galaxy install -r provisioning/requirements.yml -p provisioning/roles
```

**Comment by int-ua** [comment]
I thought it's not implemented yet since #505 is still open. I'll try that later, thanks.

**Comment by geerlingguy** [comment]
@int-ua - `-p` overrides the role install directory from the default. Closing this as a dupe of #505, which will make this a moot point.
